identity/client/e2e/utils/waitForItemInList.ts

```diff
@@ -33,7 +33,7 @@ export const waitForItemInList = async (
           emptyStateLocator?.waitFor(),
         ]);
       } else {
-        await page.getByRole("cell").first().waitFor();
+        page.getByRole("cell").filter({ hasText: /.+/ }).first().waitFor();
```

Comment:
I was getting some flakyness in when the table is actually loaded so I added another "layer" of failsafe to wait to have a table cell with text inside. Tested it and it seems to work well.